### ISSUE TITLE
Claw-Capture of spawned vessel -> failed identification for VesselGroupParam (sometimes)

### ISSUE THREAD

**Issue by GrJo** [jrossignol/ContractConfigurator#235]
In some cases, the claw-capture is like a docking (refocus etc, and everything is cool) and sometimes only like a small camera shift to indicate center-of-mass change (and then vessel identification does NOT work).

Secondary issue / hint for debugging in the COM-shift capture case: After capture, the navball does not switch to orbital mode, stays on target-relative mode and displays a steadily rising velocity (about ~1m/s², exact rate depends on docking velocity: if i capture with 2m/s relative velocity, it rises faster). Could be dependent on what part I grabbed, but i have seen it happen when attaching to RGUs, science bays and stack RCS tanks.
I have never seen this issue when using the claw in any stock recovery missions or elsewhere. So it might have something to do with the way you spawn the vessels.

Can be reproduced with the contract pack: [here](https://www.dropbox.com/s/h3tgp2blxgt6e8p/RescueAndRecovery.zip?dl=0)
Testing save (use quickload to jump to 10s before capture. Attention: there are two active contracts): [Link] (https://www.dropbox.com/s/73d966qbvsxncme/test.zip?dl=0)

**Comment by jrossignol** [comment]
Can you provide a KSP.log file?

**Comment by GrJo** [comment]
I watched the log. No exceptions or warnings on capture, just the "Joint established" line from the claw, and then something unrelated, using "VERBOSE" settings for CC. This is why I provided the contract + savegame. In that savegame, the bug happens reliably for me - have you tried reproducing it?

**Comment by jrossignol** [comment]
No luck reproducing it - I have issues with your save, but I assume that it's related to differences in mods, it seems to just die trying to go on rails:
```
[EXC 20:08:59.139] NullReferenceException
	UnityEngine.Transform.TransformDirection (Vector3 direction)
	ActiveJoint.getControlOrt (Vector3 refAxis, PartSpaceMode mode)
	ActiveJoint.applyCoordsUpdate ()
	ActiveJoint.onPartPack (.Part p)
	EventData`1[Part].Fire (.Part data)
	Part.Pack ()
	Vessel.GoOnRails ()
	HyperEdit.Model.OrbitEditor.SetOrbit (.Vessel vessel, .Orbit newOrbit)
	HyperEdit.Model.LanderAttachment.FixedUpdate ()
```
I'll try reproducing it some more with my own game and see if I can get anywhere.

**Comment by GrJo** [comment]
Strange. So the capture from the quicksave works for you? The only mods besides CC I'm using in my Dev KSP are HyperEdit, TempGaugeKiller, StockBugModuleFixes, KAC and WaypointManager. 

However, I just encountered the "capture without refocus, plus Navball mode not switching" problem in a situation not related to CC: It happened when I captured an asteroid. Will try to reproduce it in a 100% stock environment and post a bug report for the base game.
